### PR TITLE
rubocop inheritance

### PR DESCRIPTION
**Pull request by lukaselmer**
inheritance from global rubocop config

### CODE REVIEW

**Comment by sihu** [comment]
👍 nice one. i think we could discuss, what happens if the rubocop_base.yml changes. As far as i get it right now we could have new rubocop issues. But so far i also like the fact to be able to manage rubocop globally.

File: templates/.gitignore [hunk 1]
```diff
@@ -16,3 +16,4 @@
 /app/assets/javascripts/**/*.js.map
 /app/assets/javascripts/**/*.js
 .tsdrc
+.rubocop-https---raw-githubusercontent-com-renuo-rails-application-setup-guide-master-templates--rubocop-base-yml
```
Comment (sihu) [inline templates/.gitignore]: is this file generated? i couldn't find it!

Comment (lukaselmer) [inline templates/.gitignore]: yep, it is as soon as rubocop is configured this way and runs for the first time

**Comment by lukaselmer** [comment]
> As far as i get it right now we could have new rubocop issues (on CI) with a before working CI? But so far i also like the fact to be able to manage rubocop globally.

This is true. Especially if rubocop is upgraded. Having a global config file we can globally disable some rules here, so that we can upgrade rubocop, and the CI does not fail :)

But for sure, we have to be very careful when changing the rubocop file.

**Comment by sihu** [comment]
as discussed. it totally makes sense and it should fail if changes procduce problems.

**Pull request merged**